browser-switch/src/main/java/com/braintreepayments/browserswitch/BrowserSwitchResult.java

```diff
@@ -18,13 +21,14 @@
     public static final int STATUS_CANCELED = 2;
     public static final int STATUS_ERROR = 3;
 
-    BrowserSwitchResult(@BrowserSwitchStatus int status) {
-        this(status, null);
+    BrowserSwitchResult(@BrowserSwitchStatus int status, String errorMessage) {
```

Comment:
Why do we need this new `errorMessage` param here? I see we're passing in `null` for it in the `deliverResult()` function above.

Reply:
Yeah `deliverResult()` currently won't produce an error. An error message can be produced when a `start()` method is called on `BrowserSwitchClient` and, for example, a merchant app is not properly configured for BrowserSwitch because `BrowserSwitchActivity` is missing in the AndroidManifest.xml.